Explain better how to set multiple repos for `setup-r`

msberends opened the pull request:
I had thought this ought to work:

```yaml
- uses: r-lib/actions/setup-r@v2
  with:
    r-version: ${{ matrix.config.r }}
    use-public-rspm: true
    extra-repositories:
      - 'https://myrepo1.r-universe.dev'
      - 'https://myrepo2.r-universe.dev'
```

But it doesn't - it should apparently be:

```yaml
- uses: r-lib/actions/setup-r@v2
  with:
    r-version: ${{ matrix.config.r }}
    use-public-rspm: true
    extra-repositories: 'https://myrepo1.r-universe.dev, https://myrepo2.r-universe.dev'
```

This was not clear from the documentation, so I updated that.

Still think that the element should be pasted together internally to allow common YAML language like my first example.

msberends committed:
Update action.yml

gaborcsardi commented:
> Still think that the element should be pasted together internally to allow common YAML language like my first example.

Sure, a PR is welcome for that. 😄

gaborcsardi commented on setup-r/action.yml:
```diff
@@ -65,7 +65,7 @@ inputs:
     default: false
   extra-repositories:
-    description: 'One or more extra CRAN-like repositories to include in the `repos` global option'
+    description: 'One or more extra CRAN-like repositories to include in the `repos` global option. This must be a single character; multiple repos can be set using a comma: "https://repo1.org, https://repo2.org".'
```
Thanks! Can you please change this to avoid the very long line?

msberends replied:
Absolutely - I'll update it

msberends commented:
> > Still think that the element should be pasted together internally to allow common YAML language like my first example.
> 
> Sure, a PR is welcome for that. 😄

Good point, I'll try to familiarise myself with `.ts` files!

msberends committed:
Update action.yml

msberends committed:
Update installer.ts

msberends commented:
I believe this should do it, but please check critically, I've never worked in TypeScript before.

Should I also update the `README.md` or will that happen automatically?

gaborcsardi committed:
Merged origin/v2-branch into msberends-patch-1

gaborcsardi committed:
Fix typescript type error

extra_repositories is a string and getMultilineInput
returns a vector of strings.

gaborcsardi committed:
Still allow space separated repositories

gaborcsardi committed:
Update docs

gaborcsardi committed:
Transpile new setup-r

gaborcsardi committed:
Merge branch 'v2-branch' into patch-1

gaborcsardi committed:
Merge branch 'v2-branch' into patch-1

gaborcsardi commented:
Thank you!

gaborcsardi committed:
Merge branch 'v2-branch' into msberends-patch-1
